pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/DownloadParticipantListRoute.java

```diff
@@ -33,7 +37,10 @@ public Object processRequest(Request request, Response response, String userId)
         }
         setResponseHeaders(response, realm + "_export.zip");
         Filterable filterable = FilterFactory.of(request);
-        return DownloadParticipantListService.createParticipantDownloadZip(filterable, params, realm, request.queryMap(), payload, response);
+        DDPInstanceDto ddpInstanceDto = ddpInstanceDao.getDDPInstanceByInstanceName(realm)
```

Comment:
You might want to do this before the call to `checkUserAccess`, since it uses realm in a query. This is standard input validation, so should happen right at the beginning of the method.